### ISSUE TITLE
access to draggabilly instance

### ISSUE THREAD

**Issue by z1haze** [ichti-git/vue-draggabilly#1]
How do you get access to the draggabilly instance in the in dragStart, etc handlers?

Normally the context of `this` inside of that handler is the draggabilly object, but i dont see anything that shows how to get it from your directive.

**Comment by ichti-git** [comment]
Yes. The callback can actually take 3 arguments, where the first is the draggabilly instance. 
onDrag: function(draggie, event, pointer) { ... }

You can try that, it seems that how I did it myself. But I'm not 100% sure.

**Comment by z1haze** [comment]
According to the docs that is not what the callbaxk receives. Can you elaborate on this? Docs show, event, pointer, and vector path or whatever it's called. Mom are the instance of the drag object I checked
